src/MCPClient/lib/clientScripts/writeDigitalObjectComponentsToArchivesSpace.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python2
+
+from argparse import ArgumentParser
+from csv import DictWriter
+from glob import iglob
+import os
+import sys
+from uuid import uuid4
+
+from main.models import ArchivesSpaceDOComponent
+
+# archivematicaCommon
+from elasticSearchFunctions import getDashboardUUID
+
+from agentarchives.archivesspace import ArchivesSpaceClient
+
+
+def main(current_location, host, port, user, passwd):
+    dashboard_uuid = getDashboardUUID()
+
+    csvfile = open(os.path.join(current_location, 'metadata', 'digital_object_components.csv'), 'w')
```

Comment:
It'd need to remain open throughout the remaining code, though I guess I could pull that into a different function.